### PR TITLE
Added guide to Data Integration with RDS for MySQL

### PR DESCRIPTION
**Pull request by lucas-emqx**

### CODE REVIEW

File: en_US/rule_engine/rule_engine_save_rds_mysql.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Open the deployment console and click on Data Integrations, then click on the MySQL tile.
+- Open the AWS console and copy the RDS endpoint address.
+- Fill out the needed information accordingly. In this tutorial, we created the database "emqx", and an "admin:password" master user.
+
+    ![EMQX Cloud - MySQL Integration Page](./_assets/rds_mysql_integration.png)
+
+## Create and Test a New Rule
+
+- Follow the "Data Integrations Configuration" and "Test" section of [this EMQX tutorial](https://docs.emqx.com/en/cloud/latest/rule_engine/rule_engine_save_mysql.html#data-integrations-configuration).
+  - Start at the second step ("Create Rule") of "Data Integrations Configuration"
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_save_rds_mysql.md:67]: Can you add a test step? Something like this https://docs.emqx.com/en/cloud/latest/rule_engine/rule_engine_save_mysql.html#test

File: en_US/rule_engine/rule_engine_save_rds_mysql.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- Open the deployment console and click on Data Integrations, then click on the MySQL tile.
+- Open the AWS console and copy the RDS endpoint address.
+- Fill out the needed information accordingly. In this tutorial, we created the database "emqx", and an "admin:password" master user.
+
+    ![EMQX Cloud - MySQL Integration Page](./_assets/rds_mysql_integration.png)
+
+## Create and Test a New Rule
+
+- Follow the "Data Integrations Configuration" and "Test" section of [this EMQX tutorial](https://docs.emqx.com/en/cloud/latest/rule_engine/rule_engine_save_mysql.html#data-integrations-configuration).
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_save_rds_mysql.md:66]: Can you provide example SQL for the rule?

**Review by zibuyu1995 — CHANGES_REQUESTED** [review]

File: en_US/rule_engine/rule_engine_save_rds_mysql.md (hunk 1 above)
Comment (lucas-emqx) [reply en_US/rule_engine/rule_engine_save_rds_mysql.md]: done!

File: en_US/rule_engine/rule_engine_save_rds_mysql.md (hunk 2 above)
Comment (lucas-emqx) [reply en_US/rule_engine/rule_engine_save_rds_mysql.md]: done!

**Pull request closed without merging**